src/medVtkInria/vtkDataManagement/vtkMetaSurfaceMesh.cxx

```diff
@@ -516,8 +517,45 @@ void vtkMetaSurfaceMesh::ReadMeshFile (const char* filename)
   {
     outputmesh->GetPointData()->AddArray (pointarray);
   }
-  
+
+  vtkIdList* idlist = vtkIdList::New();
```

Comment:
in order to simplify the future medInria-public update, could this new paragraph be in a function?